packages/telemetry-e2e/values.yaml

```diff
@@ -116,14 +116,6 @@ drogueCloud:
 ditto:
   fullnameOverride: ditto
 
-  concierge:
-    resources:
-      requests:
-        cpu: "100m"
-        memory: 256Mi
-      limits:
-        cpu: "1"
-        memory: 384Mi
```

Comment:
@kalinkostashki the "telemetry-e2e" package should not be affected by this PR.
And update of the Ditto version in this package would have to be tracked separately.

Reply:
done